README.md

```diff
@@ -7,8 +7,48 @@ The AmpliPi design is entirely open source, from the software, to the firmware,
 
 The system is self-hosted on the Raspberry Pi and is privacy-centric. By design, AmpiPi doesn’t listen to you or spy on you — it just plays your audio! The way it should be. An internet connection is only needed for some external streaming sources, such as Pandora. The Python application running on the Pi hosts a mobile-friendly website and REST API used to control the system. The website is built on top of the REST API.
 
+![High-level Schematic](docs/imgs/Signal_Diagram_with_Expansion_Out.png)
+
 Check us out on [Kickstarter](https://www.kickstarter.com/projects/micro-nova/amplipi-home-audio-system)!
 
+## Features
+
+AmpliPi can interface with many different types of digital streaming inputs and we are working on new ones all of the time. Most of the streams support can be played independently on 4 separate sources at once, check out the **Multiple?** heading. Below is the current status of our streams.
```

Comment:
This sounds confusing to me. Is it accurate to say: "Most of the streaming services supported can be played as independent digital streams on all four sources at once"?

Reply:
I like it